packages/compass-preferences-model/src/preferences-schema.ts

```diff
@@ -1027,6 +1094,22 @@ function deriveNetworkTrafficOptionState<K extends keyof AllPreferences>(
   });
 }
 
+/** Helper for deriving value/state for preferences from other preferences */
+function deriveValueFromOtherPreferences<K extends keyof AllPreferences>(
```

Comment:
Small suggestion: This name doesn't really describe what the function does differently, since the other `derive...` helpers also derive values from other preferences. 

Maybe a name like `deriveValueFromOtherPreferencesAsConjunction` or `deriveValueFromOtherPreferencesAsLogicalAnd` is better suited to explain _how_ this particular function derives the value of a preference from other preferences?